metals/src/main/scala/scala/meta/internal/metals/MetalsEnrichments.scala

```diff
@@ -822,8 +822,14 @@ object MetalsEnrichments
       else value
     }
 
-    def symbolToFullQualifiedName: String =
-      value.replaceAll("/|#", ".").stripSuffix(".")
+    def symbolToFullyQualifiedName: String =
```

Comment:
I don't know if you did that, but you might need to special handle package objects.